mep/books/models.py

```diff
@@ -462,8 +462,13 @@ def index_data(self):
 
         earliest_date = self.earliest_date()
         if earliest_date:
+            # FIXME: acceptable for partial dates? (probably)
```

Comment:
this seems acceptable, or at least I'm not sure what the alternative would be.

Reply:
Yeah, I think I already removed that fixme somewhere? (maybe in a different branch?) I think we're ok with unknowns sorting at the first, that's what they would do even with a partial date